app/services/locations/finder.rb

```diff
@@ -2,30 +2,58 @@
 
 module Locations
   class Finder
-    attr_accessor :filter_params
+    attr_accessor :filter_params, :order_by, :order_direction
 
-    def initialize(filter_params)
+    def initialize(filter_params, sort_params = {})
       self.filter_params = filter_params
+      self.order_by = (sort_params[:by] || 'title').to_sym
```

Comment:
I believe @martyn-w said that this is okay to default to sorting by title.

Reply:
Yeah, I can't see an issue with that but I flagged as a breaking change just in case.